frontend/src/components/UserMenu.vue

```diff
@@ -13,9 +13,8 @@
       ></button>
     </div>
     <div class="user-menu-btn-big-wrapper">
-      <!-- <img src="images/ubg-logo.png" class="menu-logo-big"> -->
       <button class="user-menu-btn-big" @click="$router.push({name: 'SideQuest'})">
-        <p v-html="scanIcon"></p>
+        <img :src="require('../assets/icons/sidequest.svg')" style="width:5em" />
```

Comment:
I've seen this syntax used throughout the app, but I still don't know what it means. What's the difference with `src="../assets/icons/sidequest.svg"`?

Reply:
No expert on this either.
I think we should probably change this to just put these svgs in the public folder. Was playing around with the existing stuff and keeping mostly the same behavior. Currently your suggestion wouldn't work because the local browser cannot access that path on the server. 

The difference: sidequest.svg is now not directly accessible on the server and gets served through a versioned url. The benefits? Not sure there are any for these icons. The versioned url might help with overriding cache or something?